lib/mini_racer.rb

```diff
@@ -1,14 +1,16 @@
 require "mini_racer/version"
-require "mini_racer_loader"
+require "mini_racer_loader" unless defined?(TruffleRuby)
```

Comment:
What's `mini_racer_loader`?

Reply:
The gem has a C extension to load the implementation C extension, the notes say:
```
// Load a Ruby extension like Ruby does, only with flags that:
// a) hide symbols from other extensions (RTLD_LOCAL)
// b) bind symbols tightly (RTLD_DEEPBIND, when available)
```
https://github.com/rubyjs/mini_racer/blob/746bfddfeacf671bea7d7c7dd7897a7edcc452fd/ext/mini_racer_loader/mini_racer_loader.c#L7-L9

Reply:
Maybe move this under the `unless defined?(TruffleRuby)` below